common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -100,7 +100,7 @@ innovation_inner_peace = {
 		  }
 	}
 	modifier = {
-		monthly_lifestyle_xp_gain_mult = 0.20
+		monthly_lifestyle_xp_gain_mult = 0.5
```

Comment:
It's SO OP. It's like the entire nation of geniuses. Actually, it's better because Genius gives +30%.

Reply:
I'm asking @MemeWizard why it's so cheap on the document.

Reply:
Ok will add this to next commit